_includes/footer.html

```diff
@@ -1,7 +1,12 @@
+{% capture name_components %}
+  {% if site.translations[site.lang].main.first_name %}{{ site.translations[site.lang].main.first_name }}{% endif %}
+  {% if site.translations[site.lang].main.middle_name %}{{ site.translations[site.lang].main.middle_name }}{% endif %}
+  {% if site.translations[site.lang].main.last_name %}{{ site.translations[site.lang].main.last_name }}{% endif %}
+{% endcapture %}
     {% if site.footer_fixed %}
     <footer class="fixed-bottom">
       <div class="container mt-0">
-        &copy; Copyright {{ site.time | date: '%Y' }} {% t main.first_name %} {% t main.middle_name %} {% t main.last_name %}. {% t main.footer_text %}
+        &copy; Copyright {{ name_components | split:' ' | join:' ' }}. {% t main.footer_text %}
```

Comment:
Why do you need to split and join by spaces? If it is because there are some empty spaces surging from the `capture` above, I believe this can be fixed by using `{%- -%}`, as I saw [here](https://mathieubuisson.github.io/reducing-whitespace-jekyll-html/). To be honest, I am still [struggling with this a little](https://github.com/alshedivat/al-folio/issues/1502), but since it is a small case I think it is better if we do it right.

Reply:
I'm not well versed in liquid, and the switch from {%-  t variable %} to {{ variable }} (which was necessary to fix a bug about empty names causing nul conversion errors) caused issues with the spacing with different combinations (i.e., two spaces if there was no middle name, or a space in the beginning if there was no first name, etc.). This seemed like the easiest solution at the time that didn't require further experimentation on my end.

Reply:
I now do it the way you described via commit 143b887 (for all files). Be aware, however, that that reintroduces the issue of what happens if someones has no first name: there will be an empty space in front of the middle/last name in the metadata (can't seem to find a way around that without either doing the split+join I already did or by using a whole lot of further if-conditions.